### PR TITLE
html formatting working for spanning headers

### PR DESCRIPTION
**Pull request by shannonhaughton**

### CODE REVIEW

**Comment by shannonhaughton** [comment]
fixed #95

File: R/print_to_gt.R [hunk 1]
```diff
@@ -187,6 +187,17 @@ cleaned_data_to_gt <- function(.data, tfrmt){
       locations = list(cells_body(), cells_row_groups(), cells_stub(),
                        cells_column_labels(), cells_column_spanners())
     )
+
+
+  # create list of column headers
+    rename=gt_out_final$`_boxhead`$column_label
```
[the remark this answers is not in the document]
Comment (bzkrouse) [reply R/print_to_gt.R:193]: Since rename is a dplyr function, we should avoid that name

File: tests/testthat/test-mrkdown_labels.R [hunk 2]
```diff
@@ -0,0 +1,153 @@
+test_that("markdown column labels - no spanning",{
+
+  mock_data <- tibble::tibble(rowlbl1 =c(rep("Completion Status",12),rep("Primary reason for withdrawal",28)),
```
Comment (bzkrouse) [inline tests/testthat/test-mrkdown_labels.R:3]:
```suggestion
  mock_data <- tibble(rowlbl1 =c(rep("Completion Status",12),rep("Primary reason for withdrawal",28)),
```

Comment (shannonhaughton) [reply tests/testthat/test-mrkdown_labels.R:3]: I copied this from quite a few of the other tests. Should I remove it from those?

File: vignettes/faq.Rmd [hunk 3]
```diff
@@ -14,6 +14,55 @@ knitr::opts_chunk$set(
...
 
-```{r setup}
+```{r setup, message=FALSE, warning=FALSE}
 library(tfrmt)
+library(dplyr)
+library(gt)
+```
+
+
+### How do you add a newline within a column header?
```
Comment (bzkrouse) [inline vignettes/faq.Rmd:24]: A couple of other things to mention:
1- User can also choose to use \n to get a newline
2- More generally speak to markdown formatting. Not only can we do new lines but we can have bolding and italicizing in the labels

**Review by bzkrouse — CHANGES_REQUESTED** [review]
The markdown is working well on the labels! 
Two things:
(1) I have added a couple of small suggestions. For the vignette specifically, the example of the new line is great, but I think we should frame it more generally around markdown formatting. Maybe you can include some bolding in the example to highlight the functionality.

(2) Did you try to apply markdown formatting on the group/row labels? It looks like this may be an open issue for the gt package.

File: vignettes/faq.Rmd [hunk 4]
```diff
@@ -14,6 +14,55 @@ knitr::opts_chunk$set(
...
+
+To add a newline to your column header you need to include the markdown syntax `<br/>` in your character string. An example of this can be seen below:
```
[the remark this answers is not in the document]
Comment (bzkrouse) [reply vignettes/faq.Rmd:26]: I believe this is the preferred way for markdown, even though either one works

File: R/print_to_gt.R [hunk 5]
```diff
@@ -187,6 +187,17 @@ cleaned_data_to_gt <- function(.data, tfrmt){
...
+    names(rename)=names(gt_out_final$`_data`)
```
[the remark this answers is not in the document]
Comment (shannonhaughton) [reply R/print_to_gt.R]: Done

File: vignettes/faq.Rmd [hunk 6]
```diff
@@ -14,6 +14,55 @@ knitr::opts_chunk$set(
...
+
+```{r}
+es_data <-tibble(rowlbl1 =c(rep("Completion Status",12),rep("Primary reason for withdrawal",28)),
+                 rowlbl2 =c(rep("Completed",4),rep("Prematurely Withdrawn",4),rep("Unknown",4),rep("Adverse Event",4),rep("Lost to follow-up",4),rep("Protocol violation",4),rep("Subject decided to withdraw",4),rep("Protocol Violation",4),rep("Pre-Operative Dose[1]",4),rep("Other",4)),
+                 param=c(rep(c("n","n","pct","pct"),10)),
+                 column=c(rep(c("Placebo<br/>(N=48)","Treatment<br/>(N=38)"),20)),
```
[the remark this answers is not in the document]
Comment (shannonhaughton) [reply vignettes/faq.Rmd]: Done

File: vignettes/faq.Rmd (hunk 3 above)
Comment (shannonhaughton) [reply vignettes/faq.Rmd]: Done

**Comment by shannonhaughton** [comment]
> The markdown is working well on the labels! Two things: (1) I have added a couple of small suggestions. For the vignette specifically, the example of the new line is great, but I think we should frame it more generally around markdown formatting. Maybe you can include some bolding in the example to highlight the functionality.
> 
> (2) Did you try to apply markdown formatting on the group/row labels? It looks like this may be an open issue for the gt package.


1) I've resolved your comments now, let me know if anything else still needs changing :)
2) Yes! I've been looking into this, but as you say i've found it is an open issue with gt. They have code that reassigns it back to plain text at the moment. [https://github.com/rstudio/gt/issues/694](url)

**Review by bzkrouse — APPROVED** [review]
Looks great, thank you for these updates!
I will open an issue for the formatting of the group/row labels so we don't lose track of it.

**Pull request merged**